src/services/rns/rns.processor.ts

```diff
@@ -16,16 +16,16 @@ import SoldDomain from './models/sold-domain.model'
 /**
  * Creates the transfer record and updates DomainOwner
  */
-async function registerTransfer (transactionHash: string, tokenId: string, sellerAddress: string, buyerAddress: string, logger: Logger, domainsService: RnsBaseService) {
+async function registerTransfer (transactionHash: string, tokenId: string, sellerAddress: string, buyerAddress: string, logger: Logger, domainsService: RnsBaseService): Promise<number> {
```

Comment:
maybe just call it `txHash` so that the name doesn't change across the function.

Reply:
also the line is too long (see code smells).

Reply:
We can do this.